jar-infer/nullaway-integration-test/build.gradle

```diff
@@ -32,6 +32,22 @@ dependencies {
 
 
 test {
-  maxHeapSize = "1024m"
-  jvmArgs "-Xbootclasspath/p:${configurations.errorproneJavac.asPath}"
+    maxHeapSize = "1024m"
```

Comment:
This change is necessary, but not sufficient, to get these tests running on JDK 9+ (hence we still exclude this submodule)